tools/actions/turborepo-s3-cache/src/server.ts

```diff
@@ -62,18 +63,38 @@ async function startServer() {
       const artifactId = req.params.artifactId;
       const filename = `${artifactId}.gz`;
 
-      const command = new PutObjectCommand({
-        Bucket: bucket,
-        Key: filename,
-        Body: req,
-        ContentLength: req.headers["content-length"],
+      // Create a PassThrough stream to safely pipe the request body
+      const bodyStream = new PassThrough();
+      bodyStream.on("data", chunk => {
+        console.log("Chunk received:", chunk);
```

Comment:
how many logs does this create? From the survey results I noticed devs are concerned with logs being very verbose in our builds at the moment

Reply:
It does not have an impact for devs as it's only displayed in the post setup cache job. But we can remove them